### ISSUE TITLE
About accessibility : link label field and aria-hidden="true" option

### ISSUE THREAD

**Issue by alexandrebuffet** [ndiego/icon-block#2]
Hi there!

This block is really great, but not very accessible. 

I think it lacks a "label" field when adding a link to the icon to prevent it from being "empty". And it also lacks an option to add the attribute aria-hidden="true" on the whole block when it's about decorative elements.

**Comment by ndiego** [comment]
This issue addressed in version 1.1.0, which features the addition of an "Icon label" field. This field adds an `aria-label` attribute to the svg icon.
